demo/rating/icons/themes/_vibbo.scss

```diff
@@ -0,0 +1,2 @@
+@import '../../../../themes/vibbo';
+@import '../../../../components/rating/icons/src/index.scss';
```

Comment:
Sass files does not need to specify file extension when importing

Reply:
Is changed! BTW I think that all the components that have themes in sui are importing the index in this way